themes/metal/metal.theme.bash

```diff
@@ -9,9 +9,16 @@
 
 # inspired by previous bash_it theme : cupcake
 
+
+# aws credentials support
+# based in bash profile
+#https://github.com/jrab66/aws-profile-bash-prompt
+# to set  the AWS-profile option download the aws-profile and add it to ~/.local/bin/ dir
+
+
 # Demo:
-# ┌ⓔ virtualenv 🐲🤘user @ 💻 host in 🗂️ directory on 🌵 branch {1} ↑1 ↓1 +1 •1 ⌀1 ✗
-# └❯ cd .bash-it/themes/cupcake
+# ┌ⓔ virtualenv 🐲🤘user @ 💻 host in  [ AWS-profile] 🤖   🗂️ directory on 🌵 branch {1} ↑1 ↓1 +1 •1 ⌀1 ✗
```

Comment:
This looks inaccurate - in your code, the cupcake icon is inserted before the AWS-profile brackets. There's also an extra space in the brackets that shouldn't be there...